legacy/firmware/config.h

```diff
@@ -56,6 +56,13 @@ typedef struct {
   STORAGE_BYTES(public_key, 33);
 } StorageHDNode;
 
+typedef struct {
+  uint8_t id[32];
+  uint32_t usage;
+  uint8_t seed[64];
+  bool seedCached;  // TODO: should be secbool or ok?
```

Comment:
`secbool` was undefined here, so I wasn't sure if I should use it.

Reply:
Why couldn't we include secbool? Actually, why does Session need to be declared in the header? Isn't it just an internal structure?

Reply:
Moved to config.c.